frontstage/exceptions/exceptions.py

```diff
@@ -14,3 +9,19 @@ def __init__(self, response):
 
 class JWTValidationError(Exception):
     pass
+
+
+class MissingEnvironmentVariable(Exception):
+    def __init__(self, app, logger):
+        self.app = app
+        self.logger = logger
+        self.log_missing_env_variables()
+
+    def log_missing_env_variables(self):
```

Comment:
Ok, this answers the other comment about logging all the missing variables. It would be better not to implement this logic in an exception (or to tie the exception together with logging), but will approve this as we need a wider discussion around exception handling & logging.